rb/lib/selenium/webdriver/common/websocket_connection.rb

```diff
@@ -129,8 +129,6 @@ def callback_thread(params)
           Thread.current.report_on_exception = true
 
           yield params
-        rescue *CONNECTION_ERRORS
```

Comment:
Can you please revert CONNECTION_ERRORS change? We still want to rescue it even if it no longer crashes the whole thread.

Reply:
Still there, did you forget to push?